RoaringBitmap/src/test/java/org/roaringbitmap/TestIteratorMemory.java

```diff
@@ -107,13 +107,7 @@ public static void checkMemoryTrackingIsOK() {
   public void measureBoxedIterationAllocation() {
     if (isThreadAllocatedMemorySupported(THREAD_MBEAN)) {
       long before = getThreadAllocatedBytes(THREAD_MBEAN, Thread.currentThread().getId());
-
-      Iterator<Integer> intIterator = bitmap_a.iterator();
-      long result = 0;
-      while (intIterator.hasNext()) {
-        result += intIterator.next();
-
-      }
+      long result = bitmap_a.stream().sum();
```

Comment:
This is not equivalent to the original code in case of overflow. For safety, you should have some thing like bitmap_a.stream().mapToLong(i -> i).sum();

https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8073933